geniza/corpus/templates/admin/corpus/document/snippets/mixed_inlines_fieldsets.html

```diff
@@ -0,0 +1,20 @@
+{% load admin_extras %}
+
+{% comment %}
+workaround to reposition inlines by Bertrand Bordage:
```

Comment:
ha, I know that name (from previous python/django github interactions awhile back :-) )